yktracec/src/ykllvmwrap.cc

```diff
@@ -201,8 +201,8 @@ void initLLVM(void *Unused) {
 // Load the GlobalAOTMod.
 //
 // This must only be called from getAOTMod() for correct synchronisation.
-void loadAOTMod(struct BitcodeSection &Bitcode) {
-  auto Sf = StringRef((const char *)Bitcode.data, Bitcode.len);
+void loadAOTMod(struct BitcodeSection *Bitcode) {
+  auto Sf = StringRef((const char *)Bitcode->data, Bitcode->len);
```

Comment:
I would tend to avoid doing C casts like this and use one of the C++ casts (`reinterpret_cast` / `bit_cast` / etc) as it's more explicit.

Reply:
[40a859c](https://github.com/ykjit/yk/pull/810/commits/40a859c3c4f13498375e4b242c2c6fbeec7842ee) removes the C-style cast.

This commit would stand alone.